core/state_processor.go

```diff
@@ -86,6 +91,75 @@ func (p *StateProcessor) Process(block *types.Block, statedb *state.StateDB, cfg
 		receipts = append(receipts, receipt)
 		allLogs = append(allLogs, receipt.Logs...)
 	}
+
+	// verkle transition: if the conversoin process is in progress, move
+	// N values from the MPT into the verkle tree.
+	if fdb, ok := statedb.Database().(*state.ForkingDB); ok {
+		if fdb.InTransition() {
+			// XXX overkill, just save the parent root in the forking db
+			tt := statedb.GetTrie().(*trie.TransitionTrie)
+			mpt := tt.Base()
+
+			accIt, err := statedb.Snaps().AccountIterator(mpt.Hash(), fdb.LastAccHash)
+			if err != nil {
+				return nil, nil, 0, err
+			}
+			stIt, err := statedb.Snaps().StorageIterator(mpt.Hash(), fdb.LastAccHash, fdb.LastSlotHash)
+			if err != nil {
+				return nil, nil, 0, err
+			}
+
+			// move N=500 accounts into the verkle tree, starting with the
+			// slots from the previous account.
```

Comment:
Should we skip always the first result of the iterator? Asking since `LastAccHash/LastSlotHash` I think point to the last _migrated_ value; so in theory we'd like continue with the next?